Testing/SystemTests/tests/framework/ISISIndirectBayesTest.py

```diff
@@ -226,8 +226,7 @@ def validate(self):
     def cleanup(self):
         AnalysisDataService.clear()
         filenames = ['irs26176_graphite002_QLd.lpt','irs26176_graphite002_QLd.ql1',
-                     'irs26176_graphite002_QLd.ql2','irs26176_graphite002_QLd.ql3',
-                     'irs26176_graphite002_QLd_Parameters.nxs']
```

Comment:
Does the Parameters file no longer need removing?

Reply:
There are no Parameters file left once the test finished. So clean up failed for it every time.